internal/jsutil/jsutil.go

```diff
@@ -7,19 +7,20 @@ import (
 )
 
 var (
-	RuntimeContext      = js.Global().Get("context")
-	Binding             = js.Global().Get("context").Get("binding")
-	ObjectClass         = js.Global().Get("Object")
-	PromiseClass        = js.Global().Get("Promise")
-	RequestClass        = js.Global().Get("Request")
-	ResponseClass       = js.Global().Get("Response")
-	HeadersClass        = js.Global().Get("Headers")
-	ArrayClass          = js.Global().Get("Array")
-	Uint8ArrayClass     = js.Global().Get("Uint8Array")
-	ErrorClass          = js.Global().Get("Error")
-	ReadableStreamClass = js.Global().Get("ReadableStream")
-	DateClass           = js.Global().Get("Date")
-	Null                = js.ValueOf(nil)
+	RuntimeContext         = js.Global().Get("context")
+	Binding                = js.Global().Get("context").Get("binding")
+	ObjectClass            = js.Global().Get("Object")
+	PromiseClass           = js.Global().Get("Promise")
+	RequestClass           = js.Global().Get("Request")
+	ResponseClass          = js.Global().Get("Response")
+	HeadersClass           = js.Global().Get("Headers")
+	ArrayClass             = js.Global().Get("Array")
+	Uint8ArrayClass        = js.Global().Get("Uint8Array")
+	ErrorClass             = js.Global().Get("Error")
+	ReadableStreamClass    = js.Global().Get("ReadableStream")
+	FixedLengthStreamClass = js.Global().Get("FixedLengthStream")
```

Comment:
`FixedLengthStream` is an original feature of Cloudflare.
* https://developers.cloudflare.com/workers/runtime-apis/streams/transformstream/#fixedlengthstream

So once we start supporting another runtime (such as Deno), we need to check its existence.
However, since `syumai/workers` currently only supports Cloudflare Workers, it's OK.